wallet/src/de/schildbach/wallet/ui/VerifySeedConfirmFragment.kt

```diff
@@ -61,7 +61,7 @@ class VerifySeedConfirmFragment : VerifySeedBaseFragment() {
         view.findViewById<Toolbar>(R.id.toolbar).title = getString(R.string.verify)
 
         if (arguments?.containsKey("seed")!!) {
-            words.addAll(arguments!!.getStringArray("seed"))
+            words.addAll(arguments!!.getStringArray("seed")!!)
```

Comment:
not sure why this became necessary to add `!!`.  Is there a better way?

perhaps getStringArray was changed to be able to return null.